src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html

```diff
@@ -292,14 +293,46 @@ <h5>MATCHES</h5>
                   </a>
                 </div>
                 <div id="match-accordion-item-{{match.control_number}}" class="panel-collapse collapse" ng-class="{ 'in': match.control_number == vm.selectedBestMatch }">
-                  <div class="panel-body">
-                    <p ng-if="match.authors" class="authors">
-                      <span ng-repeat="author in match.authors">{{author.full_name}}
-                        <span ng-if="!$last"> ; </span>
-                      </span>
-                    </p>
-                    <p ng-if="match.abstract">{{match.abstract}}</p>
-                  </div>
+                  <div class="panel-body matches">
+                    <div ng-if="match.authors" class="authors">
+                      <span ng-repeat="author in match.authors">{{author.full_name}} <span ng-if="!$last"> ; </span></span>
+                      ({{vm.record.metadata.authors.length}} Authors)
+                    </div>
+                    <dl class="dl-horizontal">
+                      <dt ng-if="match.publication_info">Published In</dt>
+                      <dd>
+                        <span ng-repeat="pub_info in match.publication_info">
+                          <span ng-if="pub_info.journal_title">{{pub_info.journal_title}}</span>
```

Comment:
There are two issues with that approach:
- I want to display the publication-info in a certain sequence (first the journal_title, then journal_volume, year, and so on). I am not sure if this can be done using ng-repeat

- There's a field in `pub_info` (`pubinfo_freetext`) that is not needed to be displayed

Reply:
ok, as long as you don't display all, this approach is OK.